src/services/helpers.ts

```diff
@@ -30,3 +30,8 @@ export const moveToEnd = <T>(array: T[], index: number): T[] => {
   newDocuments.push(currDocument)
   return newDocuments
 }
+
+// fix ios issue for Django, that requires trailing slash in request url https://github.com/square/retrofit/issues/1037
+export const addTrailingSlashToUrl = (url: string): string => {
+  return url.lastIndexOf('/') !== -1 ? url : `${url}/`
```

Comment:
I don't think this works as expected. `'/asdf'.lastIndexOf('/')` also returns a value different to `-1` but has no trailing slash